### ISSUE TITLE
misnotation for HMCEloss

### ISSUE THREAD

**Issue by Astraightrain** [salesforce/hierarchicalContrastiveLearning#8]
Hi, I wanna use Himulconloss for my task. But I confused which statement is right choice for max lower layer loss.
You defined Lpair loss without (-) value. So in order to take your statement in paper,  if i taken max(lower_layer_loss), it would be minimum loss of lower layer.
but in your implementation,  i thought you wanna take maximum loss of lower layer multipliedwith minus value.

**Comment by FrankJY** [comment]
I also found this issue
layer_loss = torch.max(max_loss_lower_layer.to(layer_loss.device), layer_loss)
the code here may be
layer_loss = torch.min(min_loss_lower_layer.to(layer_loss.device), layer_loss)
